README.md

```diff
@@ -343,15 +71,27 @@ These are frontend options.
 
   Choose target to emit:
       --EmitONNXBasic - Ingest ONNX and emit the basic ONNX operations without inferred shapes.
-      --EmitONNXIR    - Ingest ONNX and emit corresponding ONNX dialect.
-      --EmitMLIR      - Lower model to MLIR built-in transformation dialect.
-      --EmitLLVMIR    - Lower model to LLVM IR (LLVM dialect).
-      --EmitLib       - Lower model to LLVM IR, emit (to file) LLVM bitcode for model, compile and link it to a shared library.
+      --EmitONNXIR   - Ingest ONNX and emit corresponding ONNX dialect.
+      --EmitMLIR     - Lower model to MLIR built-in transformation dialect.
+      --EmitLLVMIR   - Lower model to LLVM IR (LLVM dialect).
+      --EmitLib      - Lower model to LLVM IR, emit (to file) LLVM bitcode for model,
```

Comment:
I think we can just say the output, not the passes. onnx-mlir tries to choose the passes by investigating the input. 
Not sure "model" could be misleading: input can be *.onnx or *.mlir.

Reply:
That phrasing is straight out of the `onnx-mlir --help`. I like your suggestion, I changed the word "model" to "input" in both the README and the option text.